app/forms/declaration/BorderTransport.scala

```diff
@@ -29,7 +30,12 @@ case class BorderTransport(
   meansOfTransportCrossingTheBorderNationality: Option[String],
   meansOfTransportCrossingTheBorderType: String,
   meansOfTransportCrossingTheBorderIDNumber: String
-)
+) {
+
+  def extractModeValue: String = TransportCodes.extractBorderTransportValue(meansOfTransportCrossingTheBorderType)
+
+  def transportLeavingBorder: Seq[String] = Seq(extractModeValue, meansOfTransportCrossingTheBorderIDNumber)
```

Comment:
Seems a little strange to return a `Seq`

Reply:
Any proposal how to change name or implement it different?

Reply:
This is moved to the view